### PR TITLE
Standing between activating shield generators is now deadly

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Being in the path of the shield wall as it turns on now has disastrous consequences.

:cl: Kor
rscadd: Standing between activating shield generators now has disastrous consequences.
/:cl:

### CODE REVIEW

File: code/game/machinery/shieldgen.dm [hunk 1]
```diff
@@ -500,6 +500,9 @@
 	src.gen_secondary = B
 	if(A && B)
 		needs_power = 1
+	for(var/mob/living/L in get_turf(src.loc))
+		visible_message("<span class='danger'>The [src] is suddenly occupying the same space as [L]'s organs!</span>")
+		L.adjustBruteLoss(1000)
```
Comment (PKPenguin321) [inline code/game/machinery/shieldgen.dm]: brutal

Comment (duncathan) [inline code/game/machinery/shieldgen.dm]: >literally busts your torso open and takes the place of your organs
>doesn't gib

well where's the fun in that?

Comment (KorPhaeron) [inline code/game/machinery/shieldgen.dm]: I didn't want to deal with the "REMOVING FROM THE ROUND" police when science starts dragging their kills into the field gen.

If this is maintainer permission though I'll go for it.

**Comment by Anonus** [comment]
THIS IS UNWISE BUT SURE OKAY JUST DDDDDDO IT

**Comment by KorPhaeron** [comment]
If only we had dismemberment instead. 

Also I guess xenobio already has a trash can leading to space so whatever

File: code/game/machinery/shieldgen.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/machinery/shieldgen.dm]: ignore the REMOVING THEM FROM THE ROUND police and tell them to FUCK OFF

**Comment by Iamgoofball** [comment]
"removing from the round" police can fuck off

sponsored by your local lack of worker's union

**Comment by Shadowlight213** [comment]
:+1:

**Comment by Steelpoint** [comment]
I see someone may have been playing Helldivers recently.

Nice 'dangerous' chance. :+1:

**Comment by Alek2ander** [comment]
What about the other kind of shields (singulo containment)?

**Comment by CPTANT** [comment]
I do think gibbing is a bit extreme, it would also turn shield generators into a potent way of dispatching of corpses and lings.

**Comment by GunHog** [comment]
This seems too harsh in my opinion. It also prevents looting, which I also dislike. Killing the victim and flinging him aside, much as a shuttle does, seems a bit more appropriate.

**Comment by Iamgoofball** [comment]
@GunHog go away
@CPTANT go away
@Alek2ander it is indeed singulo containment

**Comment by Dorsisdwarf** [comment]
We need more fucking gibs in the round. 
There's literally only mint toxin, minibomb+ explosions, and the gibber now
RIP fueltank gibs

**Comment by Alek2ander** [comment]
@Iamgoofball Pretty sure these are the shield walls in xenobio and teleporter room. The singulo containment ones are /obj/machinery/field/containment

**Comment by GunHog** [comment]
*Singularity containment* gibbing is fine, but this is <b>Xenobiology containment</b>. Everyone knows not to be in the singulo containment field, but the Xenobiology forcefields used to be entirely harmless. That is why I feel this jump in lethality is a bit too extreme.

I have no problems with the *singularity* containment gibbing people, but not with Xenobiology, which used to be entirely harmless.

**Comment by bgobandit** [comment]
:+1:

**Comment by PKPenguin321** [comment]
who gives a shit gunhog
science happens to be the same department that can make bombs that can gib whole departments
lethality spread out here and there around the station makes this game better

**Comment by Dorsisdwarf** [comment]
Gunhog, remember that this means you can tease xenos by pretending to let them out then gibbing them in the doors when they try

**Comment by GunHog** [comment]
Why would I want to *gib* valuable test subjects? That sucks for both me AND the xeno. I do not get a shot at harvesting her organs, and she has to wait in the pen in the hopes of getting out to have some fun. Does not seem very good for either of us.

**Comment by Dorsisdwarf** [comment]
If you're raising xenoes for any reason other than to taunt them by pretending to let them go them mullering them then you're awful. Being a xeno in the pen is all about trying to get out and being killed for it.

Its like a minigame about a pow camp except with aliens

**Comment by GunHog** [comment]
So an instant gib button is better than an instant kill button, how?

**Comment by Shadowlight213** [comment]
Instant gib is better than instant kill because with instant kill people will abuse it to harvest organs easily, while instant gib prevents that.

**Comment by GunHog** [comment]
Which means this completely sucks for the alien player. Waiting a 15-20 for a chance to escape and fight only to be instagibbed? *Would you be happy with that as the Alien player?* The Scientists would not be happy about that either. No chance at organs, not even a brain to put into a Xenoborg.

I do not see a benefit for either side here. A scientist would not breed aliens JUST to gib them. That is cruel. I suppose an antag would benefit from the first month where it is still not common knowledge, and he can use it as easy body disposal.

**Comment by KorPhaeron** [comment]
And if the alien player gets out the rest of the crew dies. Someone is gonna be upset/dead/gibbed either way. The aliens arent entitled to win the zero sum game.

File: code/game/machinery/shieldgen.dm [hunk 2]
```diff
@@ -500,6 +500,9 @@
 	src.gen_secondary = B
 	if(A && B)
 		needs_power = 1
+	for(var/mob/living/L in get_turf(src.loc))
+		visible_message("<span class='danger'>The [src] is suddenly occupying the same space as [L]'s organs!</span>")
```
Comment (duncathan) [inline code/game/machinery/shieldgen.dm]: `\The [src]`

Comment (duncathan) [inline code/game/machinery/shieldgen.dm]: Might wanna make it `\the [L]` while you're at it

**Comment by KorPhaeron** [comment]
@duncathan 

Ready

**Comment by Dorsisdwarf** [comment]
Suck it gunhog. Xenobio aliens know they're just there to be killed anyway. If they dont like that they can shit eggs and ghost.

**Comment by GunHog** [comment]
Then I sincerely hopes this happens to you. Every time you get alien.

**Comment by Dorsisdwarf** [comment]
It does, because I enjoy tricking scientists into thinking I'm friendly, then eating their faces off.

**Pull request merged**